po/fr.po

```diff
@@ -28471,74 +28530,80 @@ msgstr "Traiter les répertoires récursivement"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:37
 msgid "Search only files matching PATTERN"
-msgstr ""
+msgstr "Ne chercher que dans les fichiers au motif MOTIF"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:38
 msgid "Skip files matching PATTERN"
-msgstr ""
+msgstr "Ignorer les fichiers au motif MOTIF"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:39
 msgid "Suppress error messages"
-msgstr ""
+msgstr "Masquer les messages d’erreur"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:40
 msgid "Ensure first character of actual line content lies on a tab stop"
 msgstr ""
+"S’assurer que le premier caractère correspondant au contenu est placé à un "
+"emplacement d’arrêt d’une tabulation"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:41
 msgid "Treat files as binary"
-msgstr ""
+msgstr "Traiter les fichiers comme des fichiers binaires"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:42
 msgid "Report Unix-style byte offsets"
-msgstr ""
+msgstr "Afficher  la  position relative en octets dans le style UNIX"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:44
 msgid "Invert the sense of matching"
-msgstr ""
+msgstr "Rechercher ce qui n’est pas une occurrence"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:45
 msgid "Only whole matching words"
 msgstr ""
+"Ne sélectionner que les lignes contenant des correspondances formant des "
+"mots complets"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:46
 msgid "Only whole matching lines"
-msgstr ""
+msgstr "Ne sélectionner que les correspondances exactes de ligne entière"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:47
 msgid "Ignore case (deprecated: use -i instead)"
 msgstr ""
+"Rechercher sans tenir compte de la casse (déprécié : utilisez -i en "
+"remplacement)"
 
 #: /tmp/fish/implicit/share/completions/grep.fish:48
 msgid "Treat input as a set of zero-terminated lines"
-msgstr ""
+msgstr "Traiter l’entrée en utilisant NUL comme séparateur de lignes"
```

Comment:
Inconsistent terminology for null character: “charactère null” above, now “charactère NUL”.

Reply:
My argument is just in favour of consistency – I have no qualms at all using _caractère NUL_ everywhere if that is the better option, just not a mix of both variants.